test/gateways/cams_test.exs

```diff
@@ -0,0 +1,86 @@
+defmodule Gringotts.Gateways.CamsTest do
+
+  Code.require_file "../mocks/cams_mock.exs", __DIR__
+  use ExUnit.Case ,async: false
+	alias Gringotts.{
+    CreditCard
+  }
+  alias Gringotts.Gateways.CamsMock , as: MockResponse 
+	alias Gringotts.Gateways.Cams, as: Gateway
+  
+  import Mock
+
+  @payment %CreditCard {
+    number: "4111111111111111",
+    month: 9,
+    year: 2018,
+    first_name: "Gopal",
+    last_name: "Shimpi",
+    verification_code: "123",
+    brand: "visa"
+  }
+
+  @bad_payment %CreditCard {
+    number: "411111111111111",
+    month: 9,
+    year: 2018,
+    first_name: "Gopal",
+    last_name: "Shimpi",
+    verification_code: "123",
+    brand: "visa"
+  }
+  @address %{
+    name:     "Jim Smith",
+    address1: "456 My Street",
+    address2: "Apt 1",
+    company:  "Widgets Inc",
+    city:     "Ottawa",
+    state:    "ON",
+    zip:      "K1C2N6",
+    country:  "US",
+    phone:    "(555)555-5555",
+    fax:      "(555)555-6666"
+  }
+  @options  [
+    config: %{
```

Comment:
config: %{
 +    username: "testintegrationc",
 +    password: "password9"
 + },   
keep it this way